modules/ROOT/pages/performance-and-impact.adoc

```diff
@@ -1,27 +1,67 @@
+== Data Retention and Limits
 
-* Minute Resolution for Metrics.
-* Gold and Platinum subscriptions:
- ** API analytics metrics data are retained for 90 days.
- ** Application metrics data are retained for 30 days.
+=== Gold and Platinum Subscription Limits
 
+Gold and Platinum subscribers to Anypoint Platform receive modest data retention and feature limits:
+
+Logs::
+Log limits are documented in xref:runtime-manager::viewing-log-data.adoc#log-persistence[Runtime Manager].
+
+Application Metrics::
+30 days.
+
+Enhanced API Metrics::
+30 days. The limit for data in the *Client Applications* tab is 1 hour.
+
+Basic Alerts::
+50 basic alerts.
+
+=== Titanium Subscription Limits
+
+Titanium subscribers to Anypoint Platform receive more robust data retention and feature limits:
+
+Logs::
+In raw format, your logs can occupy as much space as you receive with your Titanium subscription. Note that the storage space you receive with your Titanium subscription is used to store logs, metrics data, and search indexes. As indexes and metrics do not take up much storage capacity, the majority of the allocated storage can be used to store the raw log data.
+
+Searchable logs::
+10% of purchased storage can be allocated for searchable logs. +
+Anypoint Monitoring creates indexes on raw logs to enable you to efficiently and quickly perform log search queries. Anypoint Monitoring indexes the most recent logs for searching, up to 10% of the total purchased storage size. You can download logs to access older logs.
+
+Application Metrics::
+365 days.
+
+Enhanced API Metrics::
+30 days. The following limits also apply:
+* Data in the *Client Applications* tab is limited to 1 hour.
+* Data from *api_path_metric* and *api_client_ip_metric* tables in advanced mode of custom dashboards is limited to 1 hour.
+* Geo hash is 1 hour.
+
+Custom Metrics::
+365 days in production environments.
+// future: 30 days storage only for Sandbox - this is currently not implemented. We can either skip this part, or we can document 30 days which is future looking
+
+Advanced Alerts::
+20 per parent organization.
+
+Basic Alerts::
+50 times the number of production vCores in your organization. Note that you can also use your alert allowance for sandbox deployments, but this limit subject to the overall xref:access-management::business-groups.adoc#redistribute-resources[production vCore limit].
 
 == Screen Resolution
 
-For the best performance, use a minimum screen resolution of 1024x768.
+Use a minimum screen resolution of 1024 by 768 pixels.
 
 == Logging
 
-In Log Management, the maximum length of one line of log data is 8 KB (including date and thread metadata). Single lines that are longer than 8 KB will be truncated at the 8 KB mark.
+In Log Management, the maximum length of one line of log data is 8 KB (including date and thread metadata). Single lines that are longer than 8 KB are truncated at the 8 KB mark.
```

Comment:
Why is Log Management init. capped? Is it a label?

Reply:
```suggestion
When you manage logs, the maximum length of one line of log data is 8 KB (including date and thread metadata). Single lines that are longer than 8 KB are truncated at the 8 KB mark.
```